templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
The experiment wasn't as well-defined in that card as it generally is for our newer ones. For example, there isn't a success criteria or an expected outcome. The feelings on that card also weren't unanimous, so while I'd agree that page objects have proven to be useful, I don't think that we've shown as a team that we want to use page objects on all new projects going forward.

For projects that do want page objects, I think it would be great to define a library with a base class that provides them, rather than copying and pasting them into all new projects. To me, the threshold for Suspenders is, "will I want to copy this code into every project I make?" I think we can find a better way to share this particular piece of code than copying and pasting it.